### PR TITLE
Game wallet - UI Redesign

### PR DESCRIPTION
**Pull request by ceeriil**
### Main Component:
- Adjusted table styles in `main.tsx`
- Changed icons

### Mobile View:
- Made the token details and actions fixed at the bottom of the screen on mobile

### Check-in Page:
- Restyled check-in div so it occupies full page on the screen 

### Merge Conflict Resolution:
- Resolved merge conflicts in `index.tsx` and `nextjs/generated` (accepted all incoming changes)

### Component Restyling:
- Edited styles for `AddressMain`, `TokenRow`, and `TokenBalance` components

## Pending Issues:

1. The username in the address is currently static and needs to be made dynamic.
2. The current `README.md` doesn't match the current state of the repository.

## Work In Progress:

- Adjustments needed to improve the desktop view.
- Probably work on the other pages

### CODE REVIEW

File: .husky/pre-commit [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn lint-staged --verbose
\ No newline at end of file
+# yarn lint-staged --verbose
```
Comment (damianmarti) [inline .husky/pre-commit]: Please, we want to run the lint on every commit.

File: packages/nextjs/components/screens/Main.tsx [hunk 2]
```diff
@@ -186,122 +186,127 @@ export const Main = () => {
...
-                        value={token.name}
-                        className="w-0 h-0"
-                        onChange={t => setSelectedTokenName(t.target.value)}
+          )}
+
+          {checkedIn && !showBuy && !showSell && (
+            <>
+              <div className="rounded-xl overflow-x-scroll w-[350px]">
+                <table className="table-auto text-[0.8rem] border-seperate">
+                  <thead className="bg-[#F5F5F5]">
```
Comment (damianmarti) [inline packages/nextjs/components/screens/Main.tsx]: Maybe we can put this color on tailwind config file https://github.com/BuidlGuidl/event-wallet/blob/main/packages/nextjs/tailwind.config.js

File: packages/nextjs/generated/deployedContracts.ts [hunk 3]
```diff
@@ -4,8 +4,134 @@ const contracts = {
       name: "localhost",
       chainId: "31337",
       contracts: {
+        EventAliases: {
```
Comment (damianmarti) [inline packages/nextjs/generated/deployedContracts.ts]: Please, do not commit the changes on this file.

File: packages/nextjs/icons/MedelsIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,10 @@
+const MedelsIcon = ({ className, width, height, fill }: { className?: string; width: string; height: string, fill: string }) => {
```
Comment (damianmarti) [inline packages/nextjs/icons/MedelsIcon.tsx]: I think there is a typo here MedelsIcon -> MedalsIcon

File: .husky/pre-commit (hunk 1 above)
Comment (ceeriil) [reply .husky/pre-commit]: Thanks for the review. I’ll work on fixing the linting errors

File: packages/nextjs/scaffold.config.ts [hunk 5]
```diff
@@ -24,7 +24,7 @@ export type ScaffoldConfig = {
 
 const scaffoldConfig = {
   // The network where your DApp lives in
-  targetNetwork: chains.gnosis,
+  targetNetwork: chains.hardhat,
```
Comment (damianmarti) [inline packages/nextjs/scaffold.config.ts]: Please, do not change it, so the build is ready to deploy using gnosis.

File: packages/nextjs/components/screens/Main.tsx (hunk 2 above)
Comment (ceeriil) [reply packages/nextjs/components/screens/Main.tsx]: Yeah I think it’s better I move that to the config

File: packages/nextjs/styles/globals.css [hunk 6]
```diff
@@ -31,16 +45,40 @@ p {
...
+  background: #f5f5f5;
+  padding: 1rem 0.6rem;
+  border-radius: 0.5rem;
+}
+
+table td {
+  padding: 0.4rem 0rem;
+}
+
+/* table td:first-child {
```
Comment (damianmarti) [inline packages/nextjs/styles/globals.css]: Why is this commented on? If you don't use it, delete it please.

**Review by damianmarti — CHANGES_REQUESTED** [review]
Great work! Thanks, @ceeriil !! 

I left some small comments.

**Comment by Ifechukwudaniel** [comment]
@damianmarti   we made some changes

File: packages/nextjs/pages/index.tsx
Comment (Ifechukwudaniel) [inline packages/nextjs/pages/index.tsx:1]: @ceeriil  all color should be in the pallete

Comment (Ifechukwudaniel) [inline packages/nextjs/pages/index.tsx:1]: Change text-[#0D0D0D] and [#ffffff00]

**Comment by damianmarti** [comment]
This is looking great!! Thanks, @ceeriil and @Ifechukwudaniel !!!

Some small visual issues I found:

1. Vertical scroll on desktop

scroll

2. Scan button on mobile 

localhost_3001_ (11)

3. The icons are collapsing with the table header.

localhost_3001_ (12)

4. I think this balance box is too wide. We only show 4 decimals here, so the number shouldn't be so big. Maybe we can use a smaller width and center it.

localhost_3001_ (13)

5. We can remove the hardcoded username. The real username is already bellow it.

localhost_3001_ (14)

Thanks!!

**Comment by Ifechukwudaniel** [comment]
- [x]  Scan button on mobile
Screenshot 2023-10-12 at 5 33 11 AM

- [x]  Icons are no longer collapsing with the table header.
Screenshot 2023-10-12 at 5 31 20 AM

- [x]  Removed  username  and added punk blokies
Screenshot 2023-10-12 at 7 51 01 PM

- [x] Removed Vertical scroll on desktop
Screenshot 2023-10-12 at 11 10 09 PM

**Comment by Ifechukwudaniel** [comment]
@ceeriil has worked on  4 point  he just has to push it 

> This is looking great!! Thanks, @ceeriil and @Ifechukwudaniel !!!
> 
> Some small visual issues I found:
> 
> 1. Vertical scroll on desktop
> 
> scroll
> 
> 2. Scan button on mobile
> 
> localhost_3001_ (11)
> 
> 3. The icons are collapsing with the table header.
> 
> localhost_3001_ (12)
> 
> 4. I think this balance box is too wide. We only show 4 decimals here, so the number shouldn't be so big. Maybe we can use a smaller width and center it.
> 
> localhost_3001_ (13)
> 
> 5. We can remove the hardcoded username. The real username is already bellow it.
> 
> localhost_3001_ (14)
> 
> Thanks!!

**Comment by Ifechukwudaniel** [comment]
@damianmarti i think we have fixed all the changes 
Screenshot 2023-10-16 at 4 46 55 PM

**Comment by damianmarti** [comment]
> @damianmarti i think we have fixed all the changes Screenshot 2023-10-16 at 4 46 55 PM

Thanks!!!!

The credit box on the screenshot is still too wide. It was item 4 in the visual feedback:

> I think this balance box is too wide. We only show 4 decimals here, so the number shouldn't be so big. Maybe we can use a smaller width and center it.

What do you think about this?

**Review by damianmarti — APPROVED** [review]
Thanks!!! @ceeriil @Ifechukwudaniel

**Pull request merged**